.github/workflows/ibis-ci-2.yml

```
@@ -0,0 +1,66 @@
+          java -Dconfig=etc/config.properties \
+                --add-opens=java.base/java.nio=ALL-UNNAMED \
+                -jar ./wren-server/target/wren-server-*-executable.jar &
+      - name: Install poetry
+        run: pipx install poetry
+      - uses: actions/setup-python@v5
+        with:
+          python-version-file: ./ibis-server/pyproject.toml
+          cache: 'poetry'
+      - uses: extractions/setup-just@v2
+      - name: Cache Cargo
+        uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)